parsers/BORNHOLM_POWERLAB.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+# The arrow library is used to handle datetimes
+from datetime import datetime
+from logging import Logger, getLogger
+from typing import List, Optional
+
+from pytz import timezone
```

Comment:
We can use `zoneinfo` instead of pytz now (new python built in since python 3.9), it should have the same API/function as timezone if I remember correctly.

This is just a bonus.